ONSdigital/ras-rm-mock-eq
# Fix security issues

# What and why?
Dependabot was showing that mock-eq had 1 x critical 6 x high security alerts. I noticed that the pipfile.lock has not been updated in the repo and was showing that it was quite out of date. I have updated this.

# How to test?
Check Pifile.lock to make sure the versions are equal or greater than what's required to fix the issue. We won't know if this is fixed until the code has been merged.

# Jira
https://jira.ons.gov.uk/browse/RAS-983
